packages/addons/src/plugins/autoHeightTextarea.ts

```diff
@@ -12,16 +12,26 @@ export function createAutoHeightTextareaPlugin(): FormKitPlugin {
   const autoHeightTextareaPlugin = (node: FormKitNode) => {
     if (node.props.type !== 'textarea') return
     node.addProps(['autoHeight'])
+    node.addProps(['maxAutoHeight'])
 
     node.on('created', () => {
       const autoHeight = undefine(node.props.autoHeight)
+      const maxAutoHeight = node.props.maxAutoHeight
       if (!autoHeight || !node.context) return
       let inputElement: null | HTMLElement = null
 
+      if (maxAutoHeight && typeof maxAutoHeight !== 'number') {
+        console.warn(
+          'Invalid FormKit maxAutoHeight. <FormKit type="textarea" max-auto-height> must be a number.'
+        )
+        return
+      }
```

Comment:
We dont want to do prop checks for every prop since it would bloat the package to enormous size with all the potential error messages. Instead if the incorrect value type is provided it should just be ignored here (which it is with the `isFinite` check)
```suggestion
```